website/blog/2025-11-18-workflow-file-auto-discovery.md

```
@@ -0,0 +1,176 @@
+# Auto-discovery is purely additive
+atmos workflow deploy
+```
+
+## Examples
+
+### Basic Usage
+
+Run a workflow by name (auto-discovers the file):
+
+```bash
+$ atmos workflow deploy
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)